src/Equinox.MessageDb/Tracing.fs

```diff
@@ -19,7 +19,7 @@ type ActivityExtensions =
         act
 
     [<System.Runtime.CompilerServices.Extension>]
-    static member AddExpectedVersion(act: Activity, version: int64) =
+    static member AddExpectedVersion(act: Activity, version: int64 option) =
         act.AddTag("eqx.expected_version", version)
```

Comment:
will this render correctly with `null`, or should it become conditional?

Reply:
<img width="1710" alt="Screenshot 2022-11-21 at 9 06 45 AM" src="https://user-images.githubusercontent.com/3721521/203075048-8c13dd39-3b13-4385-a241-28ead85ba806.png">
Thanks for shouting. It correctly excludes `None` but renders the some explicitly as `Some(version)` 😬